tests/framework/db/CommandTest.php

```diff
@@ -809,17 +809,34 @@ public function testAutoRefreshTableSchema()
     {
         $db = $this->getConnection(false);
         $tableName = 'test';
+        $fkName = 'test_fk';
+
+        $this->assertNull($db->getSchema()->getTableSchema($tableName));
 
         $db->createCommand()->createTable($tableName, [
             'id' => 'pk',
+            'fk' => 'int',
             'name' => 'string',
         ])->execute();
         $initialSchema = $db->getSchema()->getTableSchema($tableName);
+        $this->assertNotNull($initialSchema);
 
         $db->createCommand()->addColumn($tableName, 'value', 'integer')->execute();
         $newSchema = $db->getSchema()->getTableSchema($tableName);
         $this->assertNotEquals($initialSchema, $newSchema);
 
+        $db->createCommand()->addForeignKey($fkName, $tableName, 'fk', $tableName, 'id')->execute();
```

Comment:
Sorry if I mislead you by pointing to `markTestSkipped()` method without futher explanations.
With this method ALL test method will be skipped, and it's not so correct now 'cause `addColumn()`, for instance, works perfect with `sqlite`, but also will not be tested.
`markTestSkipped()` is applicable if we'll split this method to several ones.